widget/embedded/src/components/ItemPicker/ItemPicker.types.ts

```diff
@@ -0,0 +1,11 @@
+export interface PropTypes {
+  onClick: () => void;
+  value: {
```

Comment:
Type can be improve by changing it something like this:

```js
  value?: {
    name: string, // this filed can be `displayName` or `title` as well.
    logo: string,
  }
```

Reply:
fixed

Reply:
Remove my comment please. I wrote the comment to you.

Reply:
I Removed it